.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
Instead of excluding, is it possible to only include sources we want instead?

Reply:
Iam not aware about a rule to do this, i can negate files inside an ignored directory but no just only include sources i want to unignore like

```
node_modules
build
packages/*
!packages/api/*
!packages/types-definitions/*
!packages/api/types/*
```

I you know a way to do it please link it and i can check it for sure.

Reply:
For the autogenerated files, they are generated by "@polkadot/typegen" package scripts, it insert eslint-disable rule in the file itself.